app/west.yml

```diff
@@ -15,7 +15,6 @@ manifest:
         # TODO: Rename once upstream offers option like `exclude` or `denylist`
         name-blacklist:
           - ci-tools
-          - hal_atmel
```

Comment:
Was this change purposeful?

Reply:
No, this snuck in accidentally. We do need to restore this for the wired SAMD21 stuff I'm working on, but shouldn't be this branch.

Reply:
I don't see this here any more. This was from a different commit to `main`.